server/pkg/database/database.go

```diff
@@ -21,6 +21,7 @@ type Database interface {
 	CreateAccount(account models.Account) (int, error)
 	GetAccount(username string) (models.Account, error)
```

Comment:
Can you rename this to `GetAccountByUsername` I think that will fit better with the other calls.